integration_test/workflow_tests/helpers.py

```diff
@@ -108,7 +107,7 @@ def create_session() -> Session:
 
 
 def get_state_machine_execution_results(
-    execution_arn, retry_interval_seconds=5, maximum_duration_seconds=60
+    execution_arn, retry_interval_seconds=5, maximum_duration_seconds=600
```

Comment:
step function takes longer for happy path tests. With the previous duration of 60s, the script times out and fails. That's why I used 600 which worked for happy path.